roles/splunk_common/tasks/main.yml

```diff
@@ -122,6 +122,10 @@
 - include_tasks: enable_dsp.yml
   when: "'dsp' in splunk and 'enable' in splunk.dsp and splunk.dsp.enable"
 
+- include_tasks: remove_clustering.yml
+  when:
+    - splunk_indexer_cluster or splunk.role == "splunk_search_head"
```

Comment:
Do you need this for `splunk.role == splunk_indexer` as well?

Reply:
I thought splunk_indexer_cluster would take care of that?